manim/mobject/geometry.py

```diff
@@ -666,12 +666,12 @@ def scale(self, factor, scale_tips=False, **kwargs):
 
         See Also
         --------
-        :meth:`~VMobject.scale`
+        :meth:`~.Mobject.scale`
```

Comment:
Hmmm Line is a Vmobject, no? So it would be better to link to VMobject.scale instead. Why did you change it?

Reply:
There is no separate `VMobject.scale`, it is directly inherited from `Mobject.scale`, which caused the "See Also" block in the rendered HTML documentation just to include `scale()` without link (and in particular, without reference to `VMobject` or `Mobject`).

Reply:
Ah good point